src/views/home/components/header/header.styles.ts

```diff
@@ -0,0 +1,64 @@
+import { createUseStyles } from "react-jss";
+
+import { Theme } from "src/styles/theme";
+
+const flexBox = { flex: 1, display: "flex" };
+const btn = (theme: Theme) => ({
+  display: "inline-flex",
+  alignItems: "center",
+  padding: [[theme.spacing(0.75), theme.spacing(1)]],
+  borderRadius: theme.spacing(1),
+  height: theme.spacing(4.5),
+  "& svg": {
+    width: 24,
+  },
+  "& p": {
+    margin: [[0, theme.spacing(1)]],
+    display: "none",
+    [theme.breakpoints.upSm]: {
+      display: "inherit",
+    },
+  },
+  "&:hover": {
+    backgroundColor: theme.palette.grey.light,
+  },
+});
+
+const useHeaderHomeStyles = createUseStyles((theme: Theme) => ({
+  header: {
+    ...flexBox,
+    fontWeight: theme.fontWeights.medium,
+    "& > *": {
```

Comment:
This kind of wildcards are very dangerous. Just apply the styles to the children that need it.